### PR TITLE
Create quiz

### PR DESCRIPTION
**Pull request by IanLoam126**

### CODE REVIEW

File: client-app/src/components/QuestionObject/#QuestionObject.jsx# [hunk 1]
```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+
```
Comment (robbie-hinman) [inline client-app/src/components/QuestionObject/#QuestionObject.jsx#]: I think that emacs is creating some funky file names - this one has '#'s around the file.

File: client-app/src/components/TeacherCreateQuizView/TeacherCreateQuizView.jsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
+import React, { Component } from 'react';
+import CreateQuestionObject from '../createQuestionObject/CreateQuestionObject';
+
+class TeacherCreateQuizView extends Component {
+	constructor(props){
+		super(props);
+		this.state = {
+			   selectValue: "10"
```
Comment (robbie-hinman) [inline client-app/src/components/TeacherCreateQuizView/TeacherCreateQuizView.jsx]: What does 'selectValue' mean here?  Maybe a more descriptive name for the variable?

File: client-app/src/components/QuestionObject/#QuestionObject.jsx# (hunk 1 above)
Comment (IanLoam126) [reply client-app/src/components/QuestionObject/#QuestionObject.jsx#]: i removed all of the extra files i could find in the latest commit

File: client-app/src/components/TeacherCreateQuizView/TeacherCreateQuizView.jsx (hunk 2 above)
Comment (IanLoam126) [reply client-app/src/components/TeacherCreateQuizView/TeacherCreateQuizView.jsx]: i changed the variable name to something more descriptive

**Pull request merged**